package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@skyscanner/stylelint-config-skyscanner",
-  "version": "7.1.0",
+  "version": "8.0.0",
```

Comment:
Something has gone weird with the last release, as this should already be in main